test/poi_tests/test_poi.py

```diff
@@ -0,0 +1,31 @@
+# run with: python -m unittest test/poi_tests/test_poi.py

```

Comment:
Um, I don't think we need this comment for each testing file. Just a description in the README is plenty.